### ISSUE TITLE
Existing/Proposed Power Dashboard Design (internal)

### ISSUE THREAD

**Issue by TrentonBush** [deployment-gap-model-education-fund/deployment-gap-model#169]
Based on Eric's [design mock ups](https://docs.google.com/presentation/d/1OVuceRIe33m8KZbuFv4ecGpKX0Yfm7N4pSE9qqAAXWM/edit#slide=id.g11aa9214416_0_4).

## Notes
* All proposed plots and tables are county-based.

## Data Mart Requirements
Input datasets:
* ISO active projects + CO2e estimates
* EIA existing plants + CO2e
* local ordinances
* state wind permitting type

Filters:
* State
* permitting type
* generation type

Basically I want the existing county data mart but restructured from wide to long format. This enables Tableau filters to work with values instead of columns.

I'll also need a separate table of local ordinances. This enables independent map overlays. This table needs a `state` column with the exact same values as the county table in order to perform synchronized filtering.
